extensions/panache/panache-common/deployment/pom.xml

```diff
@@ -33,6 +33,10 @@
             <groupId>org.ow2.asm</groupId>
             <artifactId>asm</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib-jdk8</artifactId>
```

Comment:
This is still there, but we should remove it.

Reply:
I changed the scope to `compile` so it won't get pulled in unnecessarily at least.  the `javax.validation` annotation is also supported but we're in the midst of the `javax` vs `jakartaee` annotation debate so i'm not sure what to do there as such.  My gut says to retool around the EE annotations but to wait until we have an answer for those package questions.

Reply:
Isn't that the default scope? What does it change?

Reply:
actually, you're right.  i thought it was `runtime`.  I'll switch it to `provided`.

Reply:
Do you see a problem using the `javax.validation` bits directly?  I'd have to rework a bit of code in the class visitor but maybe it just makes sense to do that now.

Reply:
What what is using that Kotlin stdlib, though?

Reply:
In this case it was only for @NotNull which came out of the kotlinc geenrated bytecode.  kotlinc will honor the java/jakarta EE annotation as well so I switched to that since we already have that in the tree.  I'll need to do a bit more work downstream but that's dependent on getting jandex to properly preserve these annotations.  In the meantime, though, this has been cleaned up.